queries.yaml

```diff
@@ -177,6 +177,23 @@ post_process:
         14: { area:  100, volume: 200000 }
         15: { area:  100, volume: 100000 }
 
+  - fn: vectordatasource.transform.build_fence
+    params:
+      base_layer: landuse
+      start_zoom: 16
+      prop_transform:
+        kind: true
+        id: true
+        source: true
+        area: true
+        min_zoom: true
+  - fn: vectordatasource.transform.drop_properties
+    params:
+      source_layer: landuse
+      start_zoom: 16
+      #end_zoom: 20
+      properties: [barrier]
+
```

Comment:
The earlier transform doesn't drop the barrier tag, so it needs to be dropped here (and for all features, not just fence features?

These could be on features at zoomed earlier than 16, change here to be 0 or the jinja to add an if macro?

Sent from my handsful device.



> On Jul 15, 2016, at 08:03, Robert Marianski <notifications@github.com> wrote:
> 
> In queries.yaml:
> 
> > +    params:
> > +      base_layer: landuse
> > +      start_zoom: 16
> > +      prop_transform:
> > +        kind: true
> > +        id: true
> > +        source: true
> > +        area: true
> > +        min_zoom: true
> > +  - fn: vectordatasource.transform.drop_properties
> > +    params:
> > +      source_layer: landuse
> > +      start_zoom: 16
> > +      #end_zoom: 20
> > +      properties: [barrier]
> > +
> We don't actually need this transform, do we? Looks like we always drop the barrier property as part of the build_fence transform.
> 
> ―
> You are receiving this because you were assigned.
> Reply to this email directly, view it on GitHub, or mute the thread.
>